overlord/ifacestate/ifacemgr_test.go

```diff
@@ -1032,3 +1033,44 @@ func (s *interfaceManagerSuite) TestSetupProfilesDevModeMultiple(c *C) {
 	c.Check(s.secBackend.SetupCalls[1].SnapInfo.Name(), Equals, siP.Name())
 	c.Check(s.secBackend.SetupCalls[1].DevMode, Equals, false)
 }
+
+func (s *interfaceManagerSuite) TestDiscardNamespace(c *C) {
+	mgr := s.manager(c)
+	si := s.mockSnap(c, sampleSnapYaml)
+
+	// Mock enough bits so that we can observe calls to snap-discard-ns
+	defer testutil.MockInternalCmdPath("")()
```

Comment:
I'm not sure we really need this MockInternalCmdPath (just yet), we could simply do "dirs.LibexecDir = cmd.BinDir` after the mockCommand was created.